source/quick-start.txt

```diff
@@ -14,6 +14,41 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Complete Chapter 1 of the
+:guides:`MongoDB Guide </atlas/account>` to set up a new
```

Comment:
I didn't know whether there's a standard way to refer to this guide (guides?). In this case, we want the user to complete just Chapter 1.